Sources/OAApplicationMode.h

```diff
@@ -62,6 +62,8 @@
 - (instancetype)initWithName:(NSString *)name stringKey:(NSString *)stringKey;
 
 - (NSDictionary *) toJson;
++ (void) saveBackupOfAppMode:(OAApplicationMode *)appMode;
```

Comment:
This is an incorrect place for this logic. Settings should be collected from backup using the OASettingsHelper (again ConfigureProfileFragment.java: 171 public void resetAppModePrefs(ApplicationMode appMode))